src/facdb.py

```diff
@@ -151,34 +202,43 @@ def by_classification():
         else:
             count_comparison(dff.loc[dff["diff"].abs() > thresh, :].sort_values("diff"))
 
-        st.header(f"Change in counts by {level}")
+        # st.header(f"Change in counts by {level}")
         dff.insert(0, level, dff.index)
-        dff = dff.sort_values("diff")
-        plotly_table(dff)
+        dff = dff.sort_values("diff", key=abs, ascending=False)
+
+        plot_diff_table(dff, ["count_old", "count_new", "diff"])
 
         """
         qc_mapped visualization
         """
-        st.header(f"Change in percentage mapped records by {level}")
+        st.header(f"Change in fraction of mapped records by {level}")
         st.write(
             """
-            Only instances where there is change in the percent
-            of mapped records and 100% of records are not mapped are reported
+            This section only reports instances where there is change in the percent
+            of mapped records
         """
         )
         dfff = qc_mapped.groupby(level).sum()
-        dfff.insert(0, level, dfff.index)
-        dfff["pctwogeom_old"] = dfff["wogeom_old"] / dfff["count_old"]
-        dfff["pctwogeom_new"] = dfff["wogeom_new"] / dfff["count_new"]
-        dfff["pctwogeom_old"] = dfff["pctwogeom_old"].round(2)
-        dfff["pctwogeom_new"] = dfff["pctwogeom_new"].round(2)
-        dfff["diff"] = dfff["pctwogeom_new"] - dfff["pctwogeom_old"]
-        dfff["diff"] = dfff["diff"].round(2)
-        dfff = dfff.loc[(dfff["diff"] != 0) & (~dfff["diff"].isna()), :]
-        dfff = dfff.sort_values("diff")
+        # dfff.insert(0, level, dfff.index)
+        dfff["pct_mapped_old"] = dfff["with_geom_old"] / dfff["count_old"]
```

Comment:
Getting a key error for "with_geom_old" when trying to run `Change in fraction of mapped records by datasource`

Reply:
you need to point it to branch 553 for it to work

Reply:
sorry that was stupid on my end